zulipterminal/ui_tools/boxes.py

```diff
@@ -468,22 +494,33 @@ def soup2markup(self, soup: Any, **state: Any) -> List[Any]:
                 markup.append(('msg_mention', element.text))
             elif element.name == 'a':
                 # LINKS
-                link = element.attrs['href']
+                link = element.attrs['href'].rstrip('/')
                 text = element.img['src'] if element.img else element.text
+                text = text.rstrip('/')
```

Comment:
The rstrips are useful to have to avoid anomalies and edge cases like https://google.com vs https://google.com/.

Reply:
This would be a useful comment :)